modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360EnricherImpl.java

```diff
@@ -91,6 +97,21 @@ private void mapExternalIdsOnSW360Release(SW360Release sw360Release, Artifact ar
         artifact.addFact(new CopyrightStatement(sw360Release.getCopyrights()));
     }
 
+    private void addLicenseFact(Optional<String> licenseList, Artifact artifact, Class<? extends ArtifactLicenseInformation> licenseClass) {
```

Comment:
Why not simply pass the constructor as function (e.g. `DeclaredLicenseInformation::new`) instead of unnecessary use of reflection?

Reply:
Lastest change reflects your proposal